openeo_processes_dask/process_implementations/cubes/load.py

```diff
@@ -86,20 +88,26 @@ def load_stac(
     temporal_extent: Optional[TemporalInterval] = None,
     bands: Optional[list[str]] = None,
     properties: Optional[dict] = None,
+    resolution: Optional[float] = None,
+    projection: Optional[Union[int, str]] = None,
+    resampling: Optional[str] = None,
```

Comment:
Is it planned to add these parameters to the `load_stac` process definition? From previous conversations, I remember, we did not want the users to worry about CRSs - but here it might be useful...

Reply:
Hi Valentina, on our side we use them to optimize the loading when possible. odc-stac allows to use COGs overviews to load the data at lower resolutions, if we find a resample_spatial process after load_stac, we pass the parameters to the load_stac internally. So, from my point of view, those parameters can be useful even if they are not exposed directly.